src/13-high-scores/test.ts

```diff
@@ -40,4 +40,10 @@ describe("High Scores Test Suite", () => {
     const input = [40];
     expect(new HighScores(input).personalTopThree).toEqual([40]);
   });
+
+  test("Personal top three from a list of scores and latest", () => {
```

Comment:
I'd rephrase the title for this test case to something like _'Personal top three keeps the order of scores'_ to better describe the issue this test case tries to cover.